.github/workflows/ci.yml

```
@@ -1,26 +1,77 @@
 name: CI
 
-on: [push]
+on: [push, pull_request]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
+    strategy:
+      matrix:
+        entry:
+          - { ruby: 2.5, gemfile: "Gemfile.min-supported" }
+          - { ruby: 2.7, gemfile: "Gemfile.latest-release", rubocop: true }
```

```suggestion
          - { ruby: 2.7, gemfile: "Gemfile.latest-release", rubocop: "rubocop" }
```
The check names for rubocop and edge are kind of weird. Would it make sense to make the value a string so the check is more readable?

---

I don't understand in what way they are weird.  Do you mean because we don't specify the value for all matrix entries?

Using a string seems misleading, since the value is being used as a boolean.  Using a string seems like it suggests that the string value will be used, rather than it being just used as a truthy value.

---

The name of the job that runs rubocop is `CI / build (2.7, Gemfile.latest-release, true)` and the one that runs edge rails `CI / build (2.7, Gemfile.rails-edge, true)`. The `true` means multiple things, and I'm wondering if a string value would result in `CI / build (2.7, Gemfile.latest-release, rubocop)` and `CI / build (2.7, Gemfile.rails-edge, edge)`.

Looking again, I guess it is only really unclear about rubocop. Feel free to ignore

---

The `CI` part is the workflow name.  The `build (2.7, Gemfile.latest-release, true)` part is the job name, which seems to be created automatically from github.  We can specify the job names explicitly, so I've pushed a commit to do that.

---

The one downside is that the job name seems to be used without expression substitution when the job is skipped and the name uses the `matrix` context object.